### ISSUE TITLE
Initial cube orientation seems wrong-ish

### ISSUE THREAD

**Issue by andrewlow** [merpig/RubiksProgram#12]
http://www.rubiksplace.com/move-notations/ - lists the defacto standard cube notation

    F (front): the face facing the solver.
    B (back): the back face.
    R (right): the right face.
    L (left): the left face.
    U (up): the upper face.
    D (down): the face opposite to the upper face.

![image](https://user-images.githubusercontent.com/2952475/178992262-5d3351d7-d742-468e-ac7b-ce21b3b15036.png)

However - when we consider this project's display of a cube and the rotations it seems the F (front) face is the bottom one (visually)
![image](https://user-images.githubusercontent.com/2952475/178992717-cd78b58d-579a-4977-a36e-272333954c98.png)

Things seem even more "wrong" when we look at U
![image](https://user-images.githubusercontent.com/2952475/178993087-bf8bfeb0-07c7-4201-ab70-9413e7ba78da.png)

L and R are similarly 'wrong' - or at least seem to be to me.

I'm guilty of not yet going digging into the code to figure out if this is easy to fix - but wanted to see if you had any interest in addressing this or not. 

For my needs - I'm looking at forking and changing some of the patterns to capture "my way" of solving the cube for demonstration purposes. However, it would be handy to have the on screen cube render match the notations.

It's a pretty cool project, thanks for sharing it.

**Comment by merpig** [comment]
I just need to change the camera view when starting the application on the y-axis by +90 degrees. My notation is based on 

- White face(middle piece) - Front(F)
- Blue face - Up(U)
- Red face - Right(R)
- Yellow face - Back(B)
- Orange face - Left(L)
- Green face - Down(D)

On my megaminx program I have the moves colored to match their face and I am planning on putting that in the rubiks program as well so that should also add more clarity.

On the topic of making updates the icons for the  forward and back buttons will be fixed soon too.

**Comment by andrewlow** [comment]
Cool - I was splunking in the code to try to figure out where (and how) three.js setup for the camera happened.

A 90deg change would help a lot - also - but this is my preference , an additional 45deg in a different axis would cause the cube to appear more like the classic sitting on table look.

If you point me at the right place in the code I will experiment - but I can keep looking around / learning three.js. Thanks for responding.

**Comment by merpig** [comment]
I rotated the cube by 90 degrees so it more closely resembles the orientation one might expect on load. Move colors have been added. The appearance of the undo and redo buttons have been fixed. On smaller view ports the undo and redo onClicks have some slight unintended effects on the menu (visually but nothing breaking), I'll get around to fixing that eventually and there's a few places where I need to clear move history when changing menu state.
